Makefile

```diff
@@ -51,8 +51,16 @@ else
 	BUILDER_IMAGE_TAG := $(shell git log -1 --pretty=%h $(BUILDER_IMAGE_DOCKERFILE))
 endif
 
+ifeq ($(shell command -v docker 2> /dev/null),)
```

Comment:
If both docker and podman are not installed, this will report an error like `podman: command not found`, this may confuse the users who don't know what podman is.
How about refactoring as following?
```
if docker installed, use docker
else if podman installed, use podman
else report error
```

Reply:
I think podman command not found is self explanatory when you Google.

Reply:
What if we did it in the opposite direction? If Podman use it, if docker use it and then the error would be about docker.

Reply:
@shawn-hurley Actually, that's probably best for an additional reason as well. If you have the podman-docker RPM installed, then the command `docker` exists, but it's really podman. If we don't check for podman first, then this change won't work for them.

Reply:
+1 @sseago didn't think of that!